inception-recommendation/src/test/java/de/tudarmstadt/ukp/inception/recommendation/exporter/RecommenderExporterTest.java

```diff
@@ -193,4 +225,8 @@ private Recommender buildRecommender(String id)
         return recommender;
     }
 
+    private static <T> Set<T> asSet(T... a)
```

Comment:
It's funny that Java doesn't offer such a method, isn't it?

Reply:
Guava has it, but we dont like guava.

Reply:
I have opened an issue with Apache Commons Collections - let's see if somebody implements it (or if one of us maybe sends them a patch)  https://issues.apache.org/jira/projects/COLLECTIONS/issues/COLLECTIONS-706